lib/logstash/filters/json.rb

```diff
@@ -46,54 +46,64 @@ class LogStash::Filters::Json < LogStash::Filters::Base
   # NOTE: if the `target` field already exists, it will be overwritten!
   config :target, :validate => :string
 
-  public
+  JSONPARSEFAILURE_TAG = "_jsonparsefailure"
+
   def register
     # Nothing to do here
-  end # def register
+  end
 
-  public
   def filter(event)
-    
-
-    @logger.debug("Running json filter", :event => event)
-
-    return unless event.include?(@source)
-
-    # TODO(colin) this field merging stuff below should be handled in Event.
+    @logger.debug? && @logger.debug("Running json filter", :event => event)
 
     source = event[@source]
+    return unless source
```

Comment:
I'm not sure if the semantic difference between `event.include?(@source)` and `event[@source]` can cause troubles when the `@source` key exists but is nil for some reason